src/app/components/header/BBCNewsBanner.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
```

Comment:
This directory should be renamed to `Header`, and the file renamed to `index.jsx`.